pkg/imager/iso/uefi.go

```diff
@@ -75,6 +81,18 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 		return err
 	}
 
+	printf("preparing loader.conf")
+
+	var loaderConfigOut bytes.Buffer
+
+	if err := template.Must(template.New("loader.conf").Parse(loaderConfigTemplate)).Execute(&loaderConfigOut, struct {
+		SecureBootEnroll string
+	}{
+		SecureBootEnroll: options.SDBootSecureBootEnrollKeys,
```

Comment:
should we do some validation of input? I don;t know how sd-boot behaves for an invalid key

Reply:
yep, I could add that, good point

Reply:
I will actually rework it to use constants, this will be way better